src/commands/txCommands/pullAll.js

```diff
@@ -1,32 +1,30 @@
 const chalk = require('chalk');
 const txlib = require('./util');
-const pullResourceTrns = require('./util/pullResourceTrns');
 const gitHelpers = require('./util/gitHelpers');
 
 const getProjectResourcesList = () =>
-	txlib
-		.getTfxResources()
+	txlib.resource
+		.list()
 		// We want to sort the array of resources so that the ALL_TRANSLATIONS_RESOURCE is
 		// downloaded first, this will allow other resources to be applied on top of
-		// any changes in that resource. Hopefully, this should prevent any changes in
-		// feature branches from being overridden by this resource
+		// any changes in that resource. This should prevent any changes in
+		// feature branches from being overwritten by this resource
 		.then(resources =>
 			resources.sort(
 				a => (a === txlib.ALL_TRANSLATIONS_RESOURCE ? -1 : 1)
 			)
 		);
 
 /**
- * Kicks off process to downloaded an individual resources trns
+ * Download individual resources trns
  * @param  {String} resource resource slug to be used when we downloaded translations
- * @return {Observable} Observable of tx process
  */
-const pullResource = resource => {
-	console.log(chalk.cyan(`Starting tx:pull for '${resource}'`));
-	return pullResourceTrns.pullResourceContent(resource).then(() => {
+const pullResource = slug => {
+	console.log(chalk.cyan(`Starting tx:pull for '${slug}'`));
+	return txlib.pullResourceContent(slug).then(() => {
```

Comment:
What does a slug look like? I think of a string like '/something'. I assume you want to distinguish it from a 'resource' maybe because it's usually a object? Not sure, but we could improve clarity/understanding here, maybe with a comment or with better naming.

Reply:
yeah I should write out terminology and probably clean up a few places where 'slug'/'resource'/'branch' are used interchangeably - that was a big source of my initial confusion as well.

overview:
- 'branch' == git branch name
- 'slug' == transifex label for a git-branch-specific 'resource': slashes replaced with underscores in order to be URL pathname friendly
- 'resource' == the full translation data associated with a branch labelled with the slug

clear as mud? Definitely needs another review to make sure the terms are used consistently, plus comment to define the terms 👍